optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -510,21 +564,26 @@ def to(self, *args, device: Optional[str] = None, dtype: Optional[torch.dtype] =
 
     @property
     def height(self) -> int:
-        height = self.unet.model.inputs[0].get_partial_shape()[2]
+        # flux transformer does not preserve info about height/width, they are knwon in vae_decoder
+        model = self.unet.model if self.unet is not None else self.vae.decoder.model
+        height = model.inputs[0].get_partial_shape()[2]
```

Comment:
the vae decoder seems to have this information while always present, let's use it by default.

Reply:
changed